srcpkgs/libcamera/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'libcamera'
+pkgname=libcamera
+version=0.2.0
+revision=1
+build_style=meson
+configure_args="-Dgstreamer=enabled -Dwerror=false -Dv4l2=true"
+hostmakedepends="pkg-config python3-yaml python3-ply python3-Jinja2"
+makedepends="openssl-devel libevent-devel gst-plugins-base1-devel gstreamer1-devel libyaml-devel"
+short_desc="Complex camera support library for Linux, Android, and ChromeOS"
+maintainer="Damiano Ognissanti <ognissanti@hotmail.se>"
```

Comment:
@damianoognissanti, are you still interested in maintaining this package?

Reply:
I'm not sure exactly how this works. I added a pull request and updated this for some time, but it never made it into the repo. My pull request was also automatically closed in august 2023, so I didn't try to update it any more (on github). Am I supposed to continue to update the template and hope that it will be added, or what is the correct way forward?

Reply:
It was not accepted back then, but it may be accepted now. In the event it is, you would be listed as the maintainer.

Reply:
@cinerea0 sure, I can be a maintainer for libcamera